device-types/DataDirect Networks/SFA200NVX2.yaml

```diff
@@ -0,0 +1,67 @@
+---
+manufacturer: DataDirect Networks
+model: SFA200NVX2
+slug: sfa200nvx2
```

Comment:
All slugs should have the manufacturer appended. In this case, please append `datadirect-networks-`